WBooks/UserRepository/UserRepository.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UserRepository.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 30/03/2022.
+//
+
+import Foundation
+import Alamofire
+
+public class UserRepository {
```

Comment:
Add internal to class and protocol.
The repository must be of internal access, since it will be managed from the ViewModel.